packages/web/src/client/components/jobs/job-submission-form.tsx

```
@@ -125,6 +146,64 @@ export function JobSubmissionForm({
+				// Check if this response is stale
+				if (activeToken.current !== currentToken) return;
+
+				console.error('Error checking playlist:', error);
+				setIsPlaylist(false);
+				setPlaylistVideos([]);
+			} finally {
+				// Check if this response is stale
+				if (activeToken.current !== currentToken) return;
+
+				setIsLoadingPlaylist(false);
+			}
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)